cluster/gce/gci/nvidia-gpus/installer.sh

```diff
@@ -62,15 +64,9 @@ check_nvidia_device() {
 }
 
 prepare_kernel_source() {
-    local kernel_git_repo="https://chromium.googlesource.com/chromiumos/third_party/kernel"
-    local kernel_version="$(uname -r)"
-    local kernel_version_stripped="$(echo ${kernel_version} | sed 's/\+//')"
-
     # Checkout the correct tag.
-    echo "Downloading kernel source at tag ${kernel_version_stripped} ..."
     pushd "${KERNEL_SRC_DIR}"
-    # TODO: Consume KERNEL SHA1 from COS image directly.
-    # git checkout "tags/v${kernel_version_stripped}"
+    git pull origin
     git checkout ${LAKITU_KERNEL_SHA1}
```

Comment:
`git checkout` will not throw error if `${LAKITU_KERNEL_SHA1}` is empty.  explicitly check if `$LAKITU_KERNEL_SHA1` is not empty or add `set -o nounset` at the top.